rust/rope/src/rope.rs

```diff
@@ -726,7 +757,7 @@ impl From<Rope> for String {
 
 impl<'a> From<&'a Rope> for String {
     fn from(r: &Rope) -> String {
-        r.slice_to_string(0, r.len())
+        r.slice_to_string(0..r.len())
```

Comment:
Lots of little examples like this that can be changed to `slice_to_string(..)` now, I might add that in this PR, what do you think @cmyr?

Reply:
ah yea, saw that earlier, I think that change makes sense where appropriate.